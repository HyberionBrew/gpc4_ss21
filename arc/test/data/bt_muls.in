2: x = 1
2: y = 1
3: x = 2
5: x = 3
5: y = 1
6: x = 9
6: y = 1
7: x = 3
9: x = 4
9: y = 1
10: y = 3
14: x = 2