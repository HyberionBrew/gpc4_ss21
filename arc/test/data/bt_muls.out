2: z = 1
3: z = 2
5: z = 3
6: z = 9
7: z = 3
9: z = 4
10: z = 12
14: z = 6
